iam/.repo-metadata.json

```diff
@@ -7,6 +7,5 @@
   "release_level": "alpha",
   "language": "python",
   "repo": "googleapis/google-cloud-python",
-  "distribution_name": "google-cloud-iam",
-  "api_id": "iamcredentials.googleapis.com",
+  "distribution_name": "google-cloud-iam"
```

Comment:
Is this a stray commit?

Reply:
This was a combination of fixing invalid json (stray comma) and removing an extra field. I can pull it out if that's preferable.

Reply:
We still have the `api_id` field in all the other `.repo-metadata.json` files:  is it unused?  I can't find that `distribution_name` is used any longer, either.

Reply:
`api_id` is relevant for some APIs and not for others. `distribution_name` should be pulling directly from setup.py or removed. I'll do that in another PR.

Reply:
How is `api_id` used?  I don't see references in this repo in any files except `.repo-metadata.json`, nor do I see it referenced in `docuploader`.

Reply:
Yes, it's not being used now. This was a metadata format we decided on to use across languages. Currently node uses the field `api_id` when generating their READMEs (see [synthtool template](https://github.com/googleapis/synthtool/blob/367efa907602f4b9b97369c0e922e23e6acaf1b0/synthtool/gcp/templates/node_library/README.md)). 

I'd also like to move our READMEs to being mostly templated, and we will start using this field at that point.